basic/src/test/resources/testRunner.yml

```diff
@@ -37,13 +37,4 @@ scenarios:
     - -Dokta.oauth2.clientSecret=VERY_SECRET
     - -Dserver.session.trackingModes=cookie
     - -Dokta.oauth2.localTokenValidation=false
-    - --batch-mode
-
-  implicit-flow-local-validation:
```

Comment:
Do we not need to specify that scenarios that shouldn't be run anymore?

Reply:
Correct! anything undefined will not run ;)

Though, that `--batch-mode` line should not have been removed

Reply:
oh, sorry it was just a line ending change, nm